pkg/util/testingjobs/mpijob/wrappers_mpijob.go

```diff
@@ -23,16 +23,17 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/utils/ptr"
 
 	"sigs.k8s.io/kueue/pkg/controller/constants"
-	"sigs.k8s.io/kueue/pkg/util/pointer"
 )
 
 // MPIJobWrapper wraps a Job.
 type MPIJobWrapper struct{ kubeflow.MPIJob }
 
 // MakeMPIJob creates a wrapper for a suspended job with a single container and parallelism=1.
 func MakeMPIJob(name, ns string) *MPIJobWrapper {
+	const replica int32 = 1
```

Comment:
I don't think this is needed. If we declare this, we don't need to use the `ptr` package. Just use `&replica`.

Reply:
I would get rid of the constant and use `ptr.To[int32](1)` where needed.

Reply:
It's right. I wanted to say what you say.

Reply:
fixed.